### PR TITLE
chore(bridge): Remove dynamic loading of available integrations

### LINKED ISSUE #7378: Cleanup of obsolete feature in Bridge
## Cleanup task

About 1.5 years ago, we introduced a feature in Keptn Bridge to dynamically load a list of available integrations. See:

image

With the ArtifactHub in place (https://artifacthub.io/packages/search?ts_query_web=keptn&sort=relevance&page=1), this feature is not needed anymore and should be removed. 

### Definition of done
- [x] In Bridge, the button **Load more information about integrations** and below label are removed:
   image
- [x] Environment variable is removed here: https://github.com/keptn/keptn/blob/master/installer/manifests/keptn/charts/control-plane/templates/core.yaml#L145
- [x] The file [integrations.html](https://github.com/keptn/get.keptn.sh/blob/master/src/integrations.html) is deleted including an update of https://github.com/keptn/get.keptn.sh/blob/master/.github/workflows/deploy_to_s3.yaml#L32
- [x] The Keptn docs is update, i.e., this page is deleted: https://keptn.sh/docs/0.14.x/bridge/load_information/

### PR DESCRIPTION
**Pull request by Brijesh-09**
## This PR
Cleanup of obsolete feature in Bridge https://github.com/keptn/keptn/issues/7378

### Related Issues
Removed the button Load more information about integrations and below label

Fixes #7378

### CODE REVIEW

File: bridge/server/app.ts [hunk 1]
```diff
@@ -26,7 +26,6 @@ const __dirname = dirname(fileURLToPath(import.meta.url));
 const apiUrl: string | undefined = process.env.API_URL;
 let apiToken: string | undefined = process.env.API_TOKEN;
 let cliDownloadLink: string | undefined = process.env.CLI_DOWNLOAD_LINK;
-let integrationsPageLink: string | undefined = process.env.INTEGRATIONS_PAGE_LINK;
```
Comment (Kirdock) [inline bridge/server/app.ts:29]: With this being removed, could you also remove the other usages? Else there will be a compile error

**Review by Kirdock — CHANGES_REQUESTED** [review]
- [x] Could you update your PR name to something like `feat(bridge): Cleanup of obsolete feature in Bridge`. Here you can see the pattern: https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md#commit-types-and-scopes
- [ ] The yarn.lock isn't updated. Could you remove `highlight.js` via `yarn remove highlight.js`?
- [ ] Seems like the dependencies `dompurify` and `marked` can also be removed.

File: bridge/server/app.ts (hunk 1 above)
Comment (Kirdock) [reply bridge/server/app.ts:29]: > where I can find the other usages

It's right below:
https://github.com/keptn/keptn/blob/60be3bfb7e9cd1d8934c95a02074d7be71585319/bridge/server/app.ts#L82-L85

**Comment by Kirdock** [comment]
With `highlight.js` being removed, this line must also be deleted:
https://github.com/keptn/keptn/blob/60be3bfb7e9cd1d8934c95a02074d7be71585319/bridge/angular.json#L45

**Comment by ermin-muratovic** [comment]
@Brijesh-09 thank you for squashing your changes. I know that there were many conflicts, because we changed a lot of code in the past days by introducing modules to Keptn's Bridge.

Please make sure to remove this two occurances of `KtbMarkdownComponent` and fix the suggestions from @Kirdock and I hope we will be able to merge your PR soon:

https://github.com/keptn/keptn/blob/60be3bfb7e9cd1d8934c95a02074d7be71585319/bridge/client/app/app.module.ts#L68

https://github.com/keptn/keptn/blob/60be3bfb7e9cd1d8934c95a02074d7be71585319/bridge/client/app/app.module.ts#L252

**Review by Kirdock — CHANGES_REQUESTED** [review]
This is now also deprecated and can be removed:
https://github.com/keptn/keptn/blob/60be3bfb7e9cd1d8934c95a02074d7be71585319/installer/manifests/keptn/templates/core.yaml#L112-L113

**Review by Kirdock — CHANGES_REQUESTED** [review]
- [ ] You committed some merge conflicts.
- [ ] You can also remove the `marked` dependency, as it is not used anymore now.
- [ ] The `yarn.lock` also needs to be adjusted.
If `yarn` is unclear: `yarn` is a package manager like `npm` and you can install it via `npm i yarn -g`. If you remove a dependency inside the package.json, always remove it via the command `yarn remove myDependency`. Yarn will maintain the `yarn.lock` file then and if you just simply remove it manually in the `package.json`, the lock file isn't adjusted. The `yarn.lock` file must not be touched on your own, yarn will do all the work here.

**Comment by Kirdock** [comment]
Closed in favor of #8344

**Pull request closed without merging**